.github/workflows/github-actions-yaml-test.yml

```diff
@@ -0,0 +1,67 @@
+name: ORFS variables.yaml tester and linter
+
+on:
+  push:
+
+jobs:
+  docs-test-job:
+    name: 'Tests for variables.yaml'
+    runs-on: ubuntu-latest
+    container:
+      image: openroad/ubuntu-cpp20
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        submodules: 'recursive'
+    - name: Run YAML script
+      run: |
+        python3 flow/scripts/generate-variable-docs.py
+    - name: Run YAML Lint
+      run: |
+        pip install yamllint==1.35.1
+        yamllint flow/scripts/variables.yaml
```

Comment:
Can we pick a yaml linting tool that, like black, for python, you can just run to fix the formatting?

I'm not picky on yaml linting rules, I just want automatic update, like black does for python.